main.md

```diff
@@ -737,29 +750,30 @@ The verifier MUST always check the SD-JWT signature to ensure that the SD-JWT
 has not been tampered with since its issuance. If the signature on the SD-JWT
 cannot be verified, the SD-JWT MUST be rejected. 
 
-## Entropy and Uniqueness of the salt
+## Entropy and Uniqueness of the random value
 
-The security model relies on the fact that the salt is not learned or guessed by
+The security model relies on the fact that the random value is not learned or guessed by
 the attacker. It is vitally important to adhere to this principle. As such, the
-salt MUST be created in such a manner that it is cryptographically random,
+random value MUST be created in such a manner that it is cryptographically random,
 long enough and has high entropy that it is not practical for the attacker to
-guess. Each salt value MUST be unique.
+guess. Each random value MUST be unique.
```

Comment:
The current wording (which was not introduced by this PR) implies that there must be a check that any two random values are different, but that is not what we want.

```suggestion
guess. A new random value MUST be chosen for each claim.
```

Reply:
good catch. paraphrased reference to `unique` in the rest of the spec too.